Drop-In/src/androidTest/java/com/braintreepayments/api/CardDetailsFragmentUITest.kt

```diff
@@ -37,17 +37,47 @@ class CardDetailsFragmentUITest {
     }
 
     @Test
-    fun whenStateIsRESUMED_expirationDateFieldIsFocused() {
+    fun whenStateIsRESUMED_andCardholderNameDisabled_expirationDateFieldIsFocused() {
+        val dropInRequest = DropInRequest()
```

Comment:
```suggestion
        val dropInRequest = DropInRequest()
        dropInRequest.cardholderNameStatus = FIELD_DISABLED
```

Reply:
This is set to disabled by default

Reply:
Right yeah. If that changes though this test could break.